Content.Shared/Roles/SharedRoleSystem.cs

```diff
@@ -202,47 +201,53 @@ private bool MindRolesUpdate(Entity<MindComponent?> ent)
             return false;
 
         //get the most important/latest mind role
-        var roleType = GetRoleTypeByTime(ent.Comp);
+        var role = GetCurrentRole(ent.Comp);
```

Comment:
For code readability you should deconstruct the returned tuple like so:
`var (protoId, locId) = GetCurrentRole(ent.Comp);`.
You don't need to name the variables like I did.